CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Enable gradient accumulation using Horovod's `backward_passes_per_step`
```

Comment:
```suggestion
- Enable gradient accumulation using Horovod's `backward_passes_per_step` ([#11911](https://github.com/PyTorchLightning/pytorch-lightning/pull/11911))
```

Reply:
Thanks, @rohitgr7 - sorry, I missed this.